src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import Image, { ImageProps } from '../Image'
+
+interface NavbarProps {
+  /** Array of JSX elements */
+  links?: JSX.Element[]
+  /** custom style applied to individual links */
+  linkStyle?: string
+  /** Logo can be text or an image */
+  logo: string | HTMLImageElement
```

Comment:
could we add a prop to display the DB logo next to the logo text? Generally, our nav bars have the DB logo and the feature title, not the masthead that you currently have.

For reference:
<img width="1105" alt="screen shot 2018-11-21 at 10 04 21 pm" src="https://user-images.githubusercontent.com/5677971/48884447-70259f00-edd9-11e8-8765-c9827a48b5ba.png">
<img width="1105" alt="screen shot 2018-11-21 at 10 04 00 pm" src="https://user-images.githubusercontent.com/5677971/48884514-aa8f3c00-edd9-11e8-90d7-25ad81d19705.png">

These are from [Cannabis Kingdom](http://graphics.dailybruin.com/cannabis-kingdom/day1.html) and [Close to Home](https://features.dailybruin.com/2017/close-to-home/), two good examples of features we've done with navbars.

However, we don't always show the DB logo, which is the case of the [alumni site](https://alumni.dailybruin.com) navbar.

Actually shit you totally reminded I wrote that navbar. You should check out [the code](https://github.com/dailybruin/alumni/tree/master/src/components/Header)! It's also mobile responsive (see `MobilePopup.tsx`). There's some interesting HTML going on, because I used a pretty new HTML tag, [details](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details). (Ideally, I think [dialog](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) would be better but the browser support isn't there yet.) Anyways, the details stuff was inspired by a talk I went to in Brooklyn last summer, some notes on it are [here](https://github.com/muan/details-on-details). Read through all that and lemme know if you have any questions; I'll help ya out! 

So I think what I'd like to see is something like this:
```suggestion
  showLogo: boolean
  title: string
```
where `title` is the text shown next to the logo (could have a better name)

Reply:
also did this, i took the logo used in the gatsby starter not sure if there's a preferred one out of the two above?